docs/core/logger.md

```diff
@@ -14,9 +14,9 @@ Logger provides an opinionated logger with output structured as JSON.
 
 ## Key features
 
-* Capture key fields from Lambda context, cold start and structures logging output as JSON
-* Log Lambda context when instructed (disabled by default)
-* Log sampling prints all logs for a percentage of invocations (disabled by default)
+* Capture key fields from Lambda context, cold start and structure logging output as JSON
+* Log Lambda event when instructed (disabled by default)
```

Comment:
Should this be `event and context`?

Reply:
I think that's captured in the point before, so it would be redundant?

Reply:
True, sorry, I was reading only the diff.

Ignore me.